### PR TITLE
UpcomingRoomCardLg component

### PR DESCRIPTION
**Pull request by sahilmob**
#2022

### CODE REVIEW

**Comment by goldyydev** [comment]
Just a few things:
@ajmnz the europeanos are offended by the MM/DD/YYYY format.
image

@sahilmob shouldn't the tags be <Tag> elements? in the future we'll need to add glows to them and emotes.

**Comment by TheOtterlord** [comment]
> @ajmnz the europeanos are offended by the MM/DD/YYYY format.

can confirm, this is really annoying 🙂

**Comment by ajmnz** [comment]
> Just a few things:
> @ajmnz the europeanos are offended by the MM/DD/YYYY format.
> image
> 
> @sahilmob shouldn't the tags be elements? in the future we'll need to add glows to them and emotes.

Can confirm, I am from Europe (12h format bothers me too), but I cannot make that dynamic on Figma. It should take into consideration the users location

**Comment by sahilmob** [comment]
@goldyydev What do you mean by elements? I used the Tag component the is mentioned under `UpcomingRoomCardLg` pre-requisites in #2022.

Also I didn't mean to offend anybody. I just wanted to be as close a possible to the design.

**Comment by ajmnz** [comment]
> Also I didn't mean to offend anybody. I just wanted to be as close a possible to the design.

You definitely didn't offend anybody! It's just that it'd be great if the date structure could be determined by the user's location

**Comment by sahilmob** [comment]
I will try to find an easy solution to format the time based on the user location

**Comment by goldyydev** [comment]
> I will try to find an easy solution to format the time based on the user location

Maybe we can display it as `MMM DD, HH:MM` (Jan 1, 00:00)

**Comment by sahilmob** [comment]
> > I will try to find an easy solution to format the time based on the user location
> 
> Maybe we can display it as `MMM DD, HH:MM` (Jan 1, 00:00)

I can change the format `MMM DD, HH:MM` if everybody is okay with it.

**Comment by Mvrs** [comment]
@sahilmob 
I noticed something... it's a small issue, but very noticeable, the header title for the card doesn't have a font weight. Just add `font-bold` and that'll give it the correct value: **700**.

**Comment by sahilmob** [comment]
> @sahilmob
> I noticed something... it's a small issue, but very noticeable, the header title for the card doesn't have a font weight. Just add `font-bold` and that'll give it the correct value: **700**.

done

File: kibbeh/src/ui/UpcomingRoomCardLg.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      <Button
+        size="small"
+        color="secondary"
+        className="mr-2 whitespace-nowrap"
+        onClick={() => setOpen((v) => !v)}
+      >
+        <span style={{ marginRight: "8px" }}>
+          <SolidCalendar />
+        </span>
+        Add to calender
```
Comment (ajmnz) [inline kibbeh/src/ui/UpcomingRoomCardLg.tsx]:
```suggestion
        Add to calendar
```

**Review by ajmnz — CHANGES_REQUESTED** [review]
While you're at it, could you change the `Tag` and `Button/small` text size from `text-xs` to `text-sm`?

**Review by ajmnz — APPROVED** [review]
Looks good!

**Pull request merged**